src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.temp.Figure;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public void setColor(String color) {
+        super.setColor(color);
+    }
```

Comment:
you don't need to override this method, remove it, you can use it and without overriding